exercises/hangman/description.md

```diff
@@ -9,8 +9,8 @@ saying "when the button is pressed increment the counter", you write
 is pressed."
 
 Implement the basic logic behind hangman using functional reactive
-programming.  You'll need to install an FRP library for this, this will
+programming. You'll need to install an FRP library for this, this will
 be described in the language/track specific files of the exercise.
 
-[Hangman]: https://en.wikipedia.org/wiki/Hangman_%28game%29
+[hangman]: https://en.wikipedia.org/wiki/Hangman_%28game%29
```

Comment:
This isn't actually linked it seems? There's only a link `[Hangman][]`

Reply:
`[Hangman][]` is a shortcut for `[Hangman][Hangman]`. IMO it should have been `[Hangman][hangman]`